set.go

```diff
@@ -77,6 +81,30 @@ var (
 	}
 )
 
+// ConcatSetType constructs a new SetDatatype which consists of a concatenation of the passed types. It panics, if the
+// nftMagic would overflow (more than 5 types)
+func ConcatSetType(types ...SetDatatype) SetDatatype {
+	if len(types) > 32/SetConcatTypeBits {
+		panic("too many type to concat")
```

Comment:
Let’s return an error instead of panicing here

Reply:
Sure. I'll added a panicing variant (MustConcatSetType), too simplify safe initializations of global variables.